features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -627,6 +627,13 @@
     <configfile finalname="${openhab.conf}/services/snmp.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/snmp</configfile>
   </feature>
 
+  <feature name="openhab-binding-sonance" description="Sonance Binding" version="${project.version}">
```

Comment:
I'm sorry, I just found one more small thing: It should be "openhab-binding-sonance1", so that we do not have any name clashes in case a Sonance binding turns up for openHAB 2.